bin/truffleruby

```diff
@@ -113,8 +139,6 @@ do
         elif [ "${val}" = "-cp" ]; then
             CP="$CP:$2"
             shift
-        elif [ "${val}" = "-cmd" ]; then
-            print_command="true"
```

Comment:
One way to keep this simple would be to add `-J` in front of all JAVA_OPTS, and then this loop processes both.